src/Timer.php

```diff
@@ -53,6 +53,20 @@ public static function stop()
     }
 
     /**
+     * get current timer and returns the elapsed time.
```

Comment:
Also, this description is wrong. "Get the elapsed time." would be a better description. The description doesn't need to explain the internal.